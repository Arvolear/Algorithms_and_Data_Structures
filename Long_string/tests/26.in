38276654308588446027800146636800829
851359967506543