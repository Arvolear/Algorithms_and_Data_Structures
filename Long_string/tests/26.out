38276654308588446028651506604307372
38276654308588446026948786669294286
32587211168419038530244531139561884440857645324147
44959424649355827871